### DISCUSSION TITLE
Git Datasource sync doesn't sync data

### DISCUSSION THREAD

**Discussion by ArxBloemJo** [netbox-community/netbox#13259] [Help Wanted!]
### NetBox version

3.5.6

### Python version

3.8

### Steps to Reproduce

1. Add new data source with following parameters:

- name: test
- type: git 
- URL: https://github.com/xxxxxxxxxxxxxx/test.git
- Username: xxxxxxx
- Password: xxxxxxxxxxx
2. Click Sync button
3. In jobs data source sync errored --> DATA = null

### Expected Behavior

Git clones the repo to netbox 

### Observed Behavior

Data 
![issue_netbox_datasource](https://github.com/netbox-community/netbox/assets/130652024/e742864a-75d9-4b6c-b422-99cb9a8ced3e)

**Comment by candlerb** [comment]
I would guess you need `https://` not `http://` to access github.

**Reply by ArxBloemJo** [reply]
Srry that was a typo on the data source its https://

**Reply by ArxBloemJo** [reply]
Is there a way to get proper logging of the auto sync ?

**Reply by abhi1693** [reply]
Try enabling logging for `netbox.core.data` and `netbox.core.jobs`

**Comment by cschar** [comment]
On the VM you are running netbox and netbox-rq service on try, seeing what your rq service is erroring out with

` sudo journalctl --unit netbox-rq.service -n 100 --no-pager`
